src/it/init-with-download-with-version-from-github/prebuild.bsh

```
@@ -0,0 +1,8 @@
+// remove cache file
+
+File cache = new File(System.getProperty("java.io.tmpdir") + "/helm-maven-plugin/github-release.json");
```

Wont this silently fail if "cache" is a folder?